R/preparedata.R

```diff
@@ -1332,8 +1332,8 @@ RearrangeRowsColumns <- function(data,
     # Keep hidden rows/columns until after sorting
     # Sort is often performed on the NET values
     data <- RemoveRowsAndOrColumns(data,
-                row.names.to.remove = row.names.to.remove,
-                column.names.to.remove = column.names.to.remove, split = split)
+                row.names.to.remove = if (!any(nzchar(row.names.to.remove))) NULL else row.names.to.remove,
+                column.names.to.remove = if (!any(nzchar(column.names.to.remove))) NULL else column.names.to.remove, split = split)
```

Comment:
Noticed when we made a Plugins yesterday that sometimes will get an error about a vector of length zero if the empty string is passed to RemoveRowsAndColumns

Reply:
Will it error if some are empty strings? E.g. "NET,,SUM" is entered by the user?

Reply:
No, its fine with "NET,,SUM" and also with " ", but its just when it has length zero.

Reply:
So `row.names.to.remove` is just a string? Why not change the condition to `row.names.to.remove == ""`?

Reply:
The default value is NULL so the zero length needs to be handled, but you're right it doesn't need to make a change for the NULL case. I've changed the condition to `isTRUE(row.names.to.remove == "")`